docs/product-assembly_business-optimizer-modifying-ER-template-IDE/src/main/asciidoc/main.adoc

```diff
@@ -0,0 +1,23 @@
+[id='assembly_optimizer-running-employee-rostering-openshift']
+
+include::product-shared-docs/document-attributes.adoc[]
+
+= Running the employee roster sample for Business Optimizer
+// Context attribute is assembly specific and enables module reuse between assemblies.
+:imagesdir: topics/product-shared-docs/images
```

Comment:
in kie-docs, we don't work with absolute imagesdir's. Those are a pain to maintain once there are many docs, because they grow out of sync.
Images must be in the same directory as the sections that use them. Don't reuse images, instead reuse sections which includes their images. The only exception to this are logo images etc, that do end up in a shared images directory.

Reply:
Do note that there might still be some content in the kie-docs that doesn't follow this "no absolute imagesdir" approach, but it has been agreed upon as a convention and all files need to adjust to it asap.

Reply:
For a good example on how that works, find all images files in here:
 https://github.com/kiegroup/kie-docs/tree/master/docs/drools-docs/src/main/asciidoc
Note that "shared/images" only contains the logo. All other images in are next to the section adoc files that use them.

Reply:
I'm afraid that the shared-doc directory system is used throughout GA docs. We can't change it for just this one document. Any changes must be decided with Vidya and Christy and we don't have the time for this now. 

The custom build system we use relies on this stuff. So Ken, please do not change it.

Reply:
To clarify, I am not saying Geoffrey is *wrong*, I am saying we can't go changing this now. I would personally prefer his proposal but this is a major change. We need to follow what we have for now.

Reply:
@mramendi The guidelines that non-shared images would reside next to the sections was agreed upon with Christy long ago, after much debate. *We need a plan and ETA to move to that.* Otherwise we're just making it harder and harder to share and reuse sections between the community reference manual and the product guides.
I 'll let this PR slide, but if we don't have a plan/ETA soon (it's been more than a year), future PR's won't...